Runtime/Scripts/Fragment/MeshSlicer.cs

```diff
@@ -275,11 +275,16 @@ public static class MeshSlicer
             var uv13 = v1.uv + s13 * (v3.uv - v1.uv);
             var uv23 = v2.uv + s23 * (v3.uv - v2.uv);
 
+            var tang13 = Vector4.Lerp(v2.tangent, v3.tangent, s13).normalized;
+            tang13.w = Mathf.Lerp(v2.tangent.w, v3.tangent.w, s13);
+            var tang23 = Vector4.Lerp(v2.tangent, v3.tangent, s23).normalized;
+            tang23.w = Mathf.Lerp(v2.tangent.w, v3.tangent.w, s23);
```

Comment:
not sure if this is correct, I was inspired by http://answers.unity.com/comments/485110/view.html